src/test/java/org/folio/rest/impl/PatronBlockLimitsAPITest.java

```diff
@@ -103,7 +101,7 @@ public void cannotCreatePatronBlockLimitWithDoubleLimitOutOfRange()
       .as(PatronBlockLimit.class);
 
     String message = getErrorMessage(actualLimit);
-    assertThat(message, is("Must be blank or a number from 0.01 to 9999.99"));
+    assertThat(message, is("Must be blank or a number from 0.0 to 9999.99"));
```

Comment:
There is only one limit with the value type `double` at the moment - "Maximum outstanding fee/fine balance". It's a monetary value, so I think `0.0` double value should be formatted as `0.00` in the response. We should probably consider copying `MonetaryValue` class from `mod-feesfines` for this purpose.